src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -218,11 +173,9 @@ void GcsActorManager::ReconstructActor(std::shared_ptr<GcsActor> actor,
     auto actor_table_data =
         std::make_shared<rpc::ActorTableData>(*mutable_actor_table_data);
     // The backend storage is reliable in the future, so the status must be ok.
-    RAY_CHECK_OK(actor_info_accessor_.AsyncUpdate(actor->GetActorID(), actor_table_data,
-                                                  [this, actor](Status status) {
-                                                    RAY_CHECK_OK(status);
-                                                    gcs_actor_scheduler_->Schedule(actor);
-                                                  }));
+    RAY_CHECK_OK(
+        actor_info_accessor_.AsyncUpdate(actor->GetActorID(), actor_table_data, nullptr));
+    gcs_actor_scheduler_->Schedule(actor);
```

Comment:
Nice, I think about it carefully. It's no problem to move 'GCS' actor' scheduler (actor); 'outside the callback function If the write order of 'AsyncUpdate' are guaranteed.

Reply:
Thanks! I thought about this too and think it's okay.